src/Traversal/Sdk/Sdk.props

```diff
@@ -31,6 +31,9 @@
     
     <!-- Targeting packs shouldn't be referenced as traversal projects don't compile. -->
     <DisableImplicitFrameworkReferences Condition="'$(DisableImplicitFrameworkReferences)' == ''">true</DisableImplicitFrameworkReferences>
+    
+    <!-- Don't automatically reference assembly packages since NoTargets don't need reference assemblies -->
+    <AutomaticallyUseReferenceAssemblyPackages Condition="'$(AutomaticallyUseReferenceAssemblyPackages)' == ''">false</AutomaticallyUseReferenceAssemblyPackages>
```

Comment:
I'm going to add a unit test real quick...

Reply:
Wow yeah good thing I did that, this is setting it after the .NET SDK so it has no affect since its set to `true` by the SDK.  I'm going to move this up to before the .NET SDK is imported.